resources/ace-dmoj.scss

```diff
@@ -0,0 +1,60 @@
+@use "vars" as *;
+
+.django-ace-widget {
+    display: inline-block;
+    position: relative;
+    > div {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+    }
+}
+
+.django-ace-widget.loading {
```

Comment:
Could turn this into proper Sass with `&.loading`.

Reply:
Done